cg/apps/demultiplex/demultiplex_api.py

```diff
@@ -7,6 +7,7 @@
 from cg.apps.demultiplex.sbatch import DEMULTIPLEX_COMMAND, DEMULTIPLEX_ERROR
 from cg.apps.slurm.slurm_api import SlurmAPI
 from cg.apps.tb import TrailblazerAPI
+from cg.constants.priority import SlurmQos
```

Comment:
I guess different HPCs can have different values? Maybe it should be configuration rather than constants?

Reply:
If we have a slurmAPI it is an option to have it be part of that?

Reply:
Yes, it would be more general if it was configuration that would be even better. At least, after this PR it will be much simpler to change it to be read from the config instead when we decide to do so.

The SlurmAPI does not seem to be written in this way. It allows the calling API to send in the priority when generating the sbatch headers. It will still have to be defined as constants somewhere.